### PR TITLE
DDF-3177 Fixes a problem where a bytesource on a closed FBOS was being passed around

### PR DESCRIPTION
**Pull request by coyotesqrl**
#### What does this PR do?
Fixes a plaintext parsing issue

#### Who is reviewing it? 
@glenhein

(please choose AT LEAST two reviewers that need to approve the PR before it can get merged)
#### Select relevant component teams: 
https://github.com/orgs/codice/teams
#### Choose 2 committers to review/merge the PR. 
(please choose ONLY two committers from below, delete the rest)
@jlcsmith
@stustison

#### How should this be tested? (List steps with links to updated documentation)
This had broken one itest in the downstream Alliance project. A build/run of that would confirm that it is resolved.

#### Any background context you want to provide?
N/A

#### What are the relevant tickets?
[DDF-3177](https://codice.atlassian.net/browse/DDF-3177)

#### Screenshots (if appropriate)
N/A

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests
#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by coyotesqrl** [comment]
build now

File: catalog/transformer/catalog-transformer-pdf/src/main/java/ddf/catalog/transformer/input/pdf/PdfInputTransformer.java [hunk 1]
```diff
@@ -169,33 +169,44 @@ private Metacard transformWithExtractors(InputStream input, String id)
                         e);
             }
 
-            ByteSource plainTextByteSource = null;
-            try (InputStream isCopy = fbos.asByteSource()
-                    .openStream()) {
+            ByteSource docByteSource = fbos.asByteSource();
+
+            try (InputStream isCopy = docByteSource.openStream();//
```
Comment (glenhein) [inline catalog/transformer/catalog-transformer-pdf/src/main/java/ddf/catalog/transformer/input/pdf/PdfInputTransformer.java]: extra comment delimiter

File: catalog/transformer/catalog-transformer-pdf/src/main/java/ddf/catalog/transformer/input/pdf/PdfInputTransformer.java [hunk 2]
```diff
@@ -169,33 +169,44 @@ private Metacard transformWithExtractors(InputStream input, String id)
...
+                    PDDocument pdfDocument = pdDocumentGenerator.apply(isCopy)) {
+
                 Parser parser = new AutoDetectParser();
-                try (TemporaryFileBackedOutputStream contentHandlerStream = new TemporaryFileBackedOutputStream()) {
+                try (InputStream metaIs = docByteSource.openStream();//
```
Comment (glenhein) [inline catalog/transformer/catalog-transformer-pdf/src/main/java/ddf/catalog/transformer/input/pdf/PdfInputTransformer.java]: extra comment delimiter

Comment (coyotesqrl) [inline catalog/transformer/catalog-transformer-pdf/src/main/java/ddf/catalog/transformer/input/pdf/PdfInputTransformer.java]: Both of these are intentional to keep the formatting from being appalling.

Comment (glenhein) [inline catalog/transformer/catalog-transformer-pdf/src/main/java/ddf/catalog/transformer/input/pdf/PdfInputTransformer.java]: ok

**Comment by coyotesqrl** [comment]
build now

**Pull request merged**